ShopLens/ImageRecognition/Classificators/WebClassificator.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using ImageRecognition.Classificators.Helpers;
+
+namespace ImageRecognition.Classificators
+{
+    // For more see https://southcentralus.dev.cognitive.microsoft.com/docs/services/450e4ba4d72542e889d93fd7b8e960de/operations/5a6264bc40d86a0ef8b2c290
+    /// <inheritdoc />
+    /// <summary>
+    /// This class classifies images by using the customvision.ai API.
+    /// </summary>
+    public class WebClassificator : IImageClassificator, IAsyncImageClassificator
```

Comment:
Would it be better if this class was called AsyncShopLensClassificator and it wouldn't be able to make http requests to cutomvision.ai? A new class called CustomVisionHTTPRequester could be made then and also an interface called IRequester and CustomVisionHTTPRequester would have uri's as const fields?

Reply:
That sounds good, but maybe it's a bit of "over-engineering" at this point? 😅